client/quantum_serverless/utils/errors.py

```diff
@@ -0,0 +1,23 @@
+"""Errors utilities."""
+from typing import Optional
+
+
+def format_err_msg(
+    message: str, code: Optional[int] = None, extra: Optional[str] = None
+):
+    """Formats error message.
+
+    Args:
+        message: main body of message
+        code: error code
```

Comment:
Can we have an error code (an string) to identify the errors (maybe we can have a json or a file with all errors categorized)?

Something like:

error_code = "AUTH1001"
message = "..."

So the error_code is a way to identify the message in some way, because the current code we are using it to pass the http status code, for example, so they are not uniques per error

Reply:
Maybe for a next iteration with the error handling... creating a new issue is enough for now

Reply:
I like the idea! Will do it here, no worries :)